job-service/docs/pages/en-us/Getting-Started-batchworker.md

```diff
@@ -4,35 +4,39 @@ title: Batch Worker Getting Started
 ---
 
 # Getting Started
-This guide provides instructions for creating, registering and testing an example Batch Worker plugin for use by the Batch Worker.
 
+This guide provides instructions for creating a Batch Worker plugin, adding the plugin to a container and then testing the Batch Worker container.
```

Comment:
I think the guide is about providing instructions on creating a Batch Worker using a Maven Archetype and testing it using the Job Service.